backend/app/main.py

```
@@ -0,0 +1,38 @@
+
+class PromptResponse(BaseModel):
+    reply: str
+    sql: str = ""
+    viz_type: str = ""
+
+@app.get("/")
+async def healthcheck():
+    return {"status": "ok"}
+
+@app.get("/healthz")
+async def healthcheck():
```

Ruff (0.12.2)

Redefinition of unused `healthcheck` from line 20

(F811)